gregoriotex/details.html

```diff
@@ -55,188 +55,492 @@
     title = 'GregorioTeX';
 </script>
 
-<h1>Detailed information on GregorioTeX</h1>
+<h1>Detailed information on Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></h1>
 
-<p>This page presents some detailed instructions on the use of GregorioTeX, starting with the most basic: best practices for creating an engraved (that is, typeset) score in GregorioTeX, then some simple options for modifying the score, and finally a precise description of the contents of a GregorioTeX file.</p>
+<p>This page presents some detailed instructions on the use of Gregorio<span class="tex">T<span class="epsilon">e</span>X</span>, starting with the most basic: best practices for creating an engraved (that is, typeset) score in Gregorio<span class="tex">T<span class="epsilon">e</span>X</span>, then some simple options for modifying the score, and finally a precise description of the contents of a Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> file.</p>
 
 <h2>Best practices for creating an engraved score</h2>
 
-<p>In order to produce a perfect result and avoid errors in placement, you will need to specify some changes in line breaks by hand. The following is the most effective and simple method:</p>
+<p>In order to produce a perfect result and avoid errors in placement, you may need to specify some changes in line breaks by hand. The following is the most effective and simple method:</p>
```

Comment:
As a bit of a perfectionist myself, I would rather use `better` than `perfect` here.